domains/transfer/Transfer.API/API.IntegrationTests/Migrations/RollbackMigrationTests.cs

```diff
@@ -0,0 +1,29 @@
+using API.Data;
+using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using API.IntegrationTests.Testcontainers;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Xunit;
+
+namespace API.IntegrationTests.Migrations;
+
+public class RollbackMigrationTests
+{
+    [Fact]
+    public async Task can_rollback_all_migrations()
+    {
+        var container = new PostgresContainer();
+        await container.InitializeAsync();
+
+        var contextOptions = new DbContextOptionsBuilder<ApplicationDbContext>().UseNpgsql(container.ConnectionString)
+            .Options;
+        var dbContext = new ApplicationDbContext(contextOptions);
+
+        var migrator = dbContext.Database.GetService<IMigrator>();
+
+        var rollbackAllMigrations = () => migrator.Migrate("0");
```

Comment:
There is a step missing where the migrations is first applied (e.g. by calling dbContext.Database.Migrate()) and then rollback to the first migration.

Reply:
Pretty important.. Added